.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,56 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[Feature Request]: "
+labels: ["needs-triage"]
```

Comment:
Also `feature-request` and probably `customer-reported`, as well as for the bug.

Reply:
Customer-reported isn't needed. The GitHub bot will figure that out and label the issue with `customer-reported` if it is.

Reply:
I'd vote for moving away from "feature-request" label since type "Feature" lets us know its a Feature.

Reply:
Then can we get rid of the "[BUG] " and "[Feature Request] " prefixes in titles? Same reason: it's noise. I can already see that from tags/type.

Reply:
I'm not opposed to removing that prefix. If we are in agreement I'll remove it.

Reply:
Especially with "[Feature Request]: " - that's 19 characters wasted in a title and could make triaging harder.